plugins/GSdx/GSRenderer.cpp

```diff
@@ -552,17 +552,41 @@ void GSRenderer::VSync(int field)
 	}
 }
 
+static bool IsTimeEqual(tm t1, tm t2)
```

Comment:
I could remove this useless bool function if I follow [Flatout's idea](https://github.com/PCSX2/pcsx2/pull/1771#issuecomment-272634750). So yeah, I'll be doing that. ;)